### PR TITLE
Components as pure function

### PR DESCRIPTION
**Pull request by atrovato**
Removes 6 on 132 warnings on front side.

```
Component should be written as a pure function  react/prefer-stateless-function
```

Cloudflare link: https://components-as-pure-function.gladys.pages.dev/

### CODE REVIEW

File: front/src/routes/forgot-password/index.js [hunk 1]
```diff
@@ -1,12 +1,68 @@
...
 class ForgotPassword extends Component {
-  render(props, {}) {
-    return <ForgotPasswordPage {...props} />;
+  forgotPassword = async e => {
+    const { forgotPasswordEmail } = this.state;
+    if (e) {
+      e.preventDefault();
+    }
+
+    if (!validateEmail(forgotPasswordEmail)) {
```
Comment (Pierre-Gilles) [inline front/src/routes/forgot-password/index.js:16]: Forgot password crash here after changes

**Review by Pierre-Gilles — CHANGES_REQUESTED** [review]
Nice! Tested the PR and the forgot password page is broken

File: front/src/routes/forgot-password/index.js (hunk 1 above)
Comment (atrovato) [reply front/src/routes/forgot-password/index.js:16]: Fixed, I changed
```javascript
import { validateEmail } from '../../utils/validateEmail';
```
to 
```javascript
import validateEmail from '../../utils/validateEmail';
```

**Review by Pierre-Gilles — APPROVED** [review]
Fixed, it works now! Thanks for the fixes 🙏

**Pull request merged**